tests/e2e/specs/miscellaneous/WorkpaseUsingParent.spec.ts

```diff
@@ -0,0 +1,99 @@
+const dashboard: Dashboard = e2eContainer.get(CLASSES.Dashboard);
+const shellExecutor: ShellExecutor = e2eContainer.get(CLASSES.ShellExecutor);
+const workspaceHandlingTests: WorkspaceHandlingTests = e2eContainer.get(CLASSES.WorkspaceHandlingTests);
+function executeArbitraryShellScript(command: string): string {
+	const output: ShellString = shellExecutor.executeCommand(command);
+	if (output.stderr.length > 0) {
+		assert.fail(output.stderr);
+	}
+	return output.stdout;
+}
+suite('Workspace using a parent test suite', function (): void {
+	loginTests.loginIntoChe();
+	let podName: string = '';
+
+	test('Create a workspace using a parent', async function (): Promise<void> {
+		const factoryUrl: string = `${BASE_TEST_CONSTANTS.TS_SELENIUM_BASE_URL}/dashboard/#https://github.com/testsfactory/parentDevfile`;
+		await dashboard.waitPage();
+		await browserTabsUtil.navigateTo(factoryUrl);
+		await workspaceHandlingTests.obtainWorkspaceNameFromStartingPage();
+		registerRunningWorkspace(WorkspaceHandlingTests.getWorkspaceName());
+		await projectAndFileTests.waitWorkspaceReadinessForCheCodeEditor();
+		// sometimes the trust dialog does not appear at first time, for avoiding this problem we send click event for activating
+		await new Workbench().click();
+		await projectAndFileTests.performTrustAuthorDialog();
+	});
+
+	test('Check cloning of the test project', async function (): Promise<void> {
+		const expectedProjectItems: string[] = ['.devfile.yaml', 'parent.yaml', 'README.md'];
+		const visibleContent: ViewSection = await new SideBarView().getContent().getSection('parentdevfile');
+
+		for (const expectedProjectItem of expectedProjectItems) {
+			const visibleItem: ViewItem | undefined = await visibleContent.findItem(expectedProjectItem);
+			expect(visibleItem).not.equal(undefined);
+		}
+	});
+
+	test('Check devfile VS Code tasks ', async function (): Promise<void> {
+		const input: QuickOpenBox | InputBox = await new Workbench().openCommandPrompt();
+		await input.setText('>Tasks: Run Task');
+		const runTaskItem: QuickPickItem | undefined = await input.findQuickPick('Tasks: Run Task');
+		await runTaskItem?.click();
+		const devFileTask: QuickPickItem | undefined = await input.findQuickPick('devfile');
+		await devFileTask?.click();
+		const firstExpectedQuickPick: QuickPickItem | undefined = await input.findQuickPick('1. This command from the devfile');
+		const secondExpectedQuickPick: QuickPickItem | undefined = await input.findQuickPick('2. This command from the parent');
+		expect(firstExpectedQuickPick).not.eqls(undefined);
+		expect(secondExpectedQuickPick).not.eqls(undefined);
+	});
+
+	test('Check expected containers in the parent POD', function (): void {
+		const getPodNameCommand: string = `${API_TEST_CONSTANTS.TS_API_TEST_KUBERNETES_COMMAND_LINE_TOOL} get pods --selector=controller.devfile.io/devworkspace_name=sample-using-parent --output jsonpath=\'{.items[0].metadata.name}\'`;
```

Comment:
can`t check test suite locally because loginToOcp() method is missing.

Reply:
Well, I supposed if we run it on CI - it will be applied approach like in the PredefinedNamespaceTest: https://main-jenkins-csb-crwqe.apps.ocp-c1.prod.psi.redhat.com/job/Testing/job/e2e/job/basic/job/typescript-tests/12044/console, and mount kubeconfig directory: `-v /home/hudson/.kube:/home/seluser/.kube:Z ` this file stores sessions with logins. Abd we do not need to do it in the test twice.  If you want to check it locally you can log in with oc client manually and run the test as well.

Reply:
it is additional manual steps to remember. method checks firstly if user already logged.
https://github.com/eclipse/che/blob/17b7a3e8e0e3138ddfadd95414a97e931e4667d0/tests/e2e/specs/miscellaneous/PredefinedNamespace.spec.ts#L38C14-L38C14
in the PredefinedNamespaceTest we have login method after refactoring

Reply:
Applied in the https://github.com/eclipse/che/pull/22544